CHANGES.md

```diff
@@ -38,15 +38,32 @@ OpenSSL 3.5
   TLS hybrid key post-quantum/classical key agreement schemes.
   *Michael Baentsch, Viktor Dukhovni, Shane Lontis and Paul Dale*
 
-* Add ML-DSA as specified in FIPS 204.
+* Add ML-DSA as specified in FIPS 204
 
   The base code was derived from BoringSSL C++ code.
   *Shane Lontis, Viktor Dukhovni and Paul Dale*
 
- * Added new API calls to enable 3rd party QUIC stacks to use the OpenSSL TLS
-   implementation.
+* Added new API calls to enable 3rd party QUIC stacks to use the OpenSSL TLS
+  implementation.
 
-   *Matt Caswell*
+  *Matt Caswell*
+
+* Added support for a new callback registration SSL_CTX_set_new_pending_conn_cb,
+  which allows for application notification of new connection SSL object
+  creation, which occurs independently of calls to SSL_accept_connection().
+  Note: QUIC objects passed through SSL callbacks should not have their state
+  mutated via calls back into the SSL api until such time as they have been
+  received via a call to SSL_accept_connection().
+
+  *Neil Horman*
+
+* Support DEFAULT keyword and '-' prefix in SSL_CTX_set1_groups_list().
+  SSL_CTX_set1_groups_list() now supports the DEFAULT keyword which sets the
+  available groups to the default selection. The '-' prefix allows the calling
+  application to remove a group from the selection.
```

Comment:
Also duplicated. Suggest redoing CHANGES.md from scratch, just manually prepend your changes to the unmodified version from master, ignoring git's attempts to "merge".
```suggestion
```